packages/js/src/classic-editor/components/metabox/index.js

```diff
@@ -0,0 +1,140 @@
+
+	return (
+		<KeywordInput
+			keyword={ focusKeyphrase }
+			displayNoKeyphraseMessage={ displayNoKeyphraseMessage }
+			displayNoKeyphraseForTrackingMessage={ displayNoKeyphraseForTrackingMessage }
+			isSEMrushIntegrationActive={ isSEMrushIntegrationActive }
+			helpLink={ focusKeyphraseInfoLink }
+			onFocusKeywordChange={ useDebounce( handleFocusKeyphraseChange ) }
+			onFocusKeyword={ pauseMarker }
+			onBlurKeyword={ startMarker }
+		/>
+	);
+};
+
+FocusKeyphraseInput.propTypes = {
+	focusKeyphraseInfoLink: PropTypes.string.isRequired,
+};
+
+/**
+ * Creates the Metabox component.
+ *
+ * @returns {JSX.Element} The Metabox.
+ */
+const Metabox = () => {
+	const settings = useSelect( select => select( EDITOR_STORE_NAME ).getPreferences() );
+	const isKeywordAnalysisActive = useSelect( select => select( EDITOR_STORE_NAME ).getIsKeywordAnalysisActive() );
+	const baseUrl = useSelect( select => select( EDITOR_STORE_NAME ).getBaseUrlFromSettings() );
+	const shoppingData = useSelect( select => select( EDITOR_STORE_NAME ).getShoppingData() );
+	const siteIconUrl = useSelect( select => select( EDITOR_STORE_NAME ).getSiteIconUrlFromSettings() );
+	const previewImageUrl = useSelect( select => select( EDITOR_STORE_NAME ).getSnippetEditorPreviewImageUrl() );
+	const analysisType = useSelect( select => select( SEO_STORE_NAME ).selectAnalysisType() );
+
+	return (
+		<Fragment>
+			<SidebarItem
+				key="warning"
+				renderPriority={ 1 }
+			>
+				<Warning />
+			</SidebarItem>
+			{ isKeywordAnalysisActive && <SidebarItem key="keyword-input" renderPriority={ 8 }>
+				<FocusKeyphraseInput focusKeyphraseInfoLink={ wpseoAdminL10n[ "shortlinks.focus_keyword_info" ] } />
+				{ ! wpseoScriptData.metabox.isPremium && <Fill name="YoastRelatedKeyphrases">
+					<SEMrushRelatedKeyphrases />
+				</Fill> }
+			</SidebarItem> }
+			<SidebarItem key="google-preview" renderPriority={ 9 }>
+				<MetaboxCollapsible
+					id={ "yoast-snippet-editor-metabox" }
+					title={ __( "Google preview", "wordpress-seo" ) } initialIsOpen={ true }
+				>
+					<GooglePreviewContainer
+						as={ SnippetEditor }
+						baseUrl={ baseUrl }
+						shoppingData={ shoppingData }
+						faviconSrc={ siteIconUrl }
+						mobileImageSrc={ previewImageUrl }
+						isTaxonomy={ analysisType === "term" }
+					/>
+				</MetaboxCollapsible>
+			</SidebarItem>
+			{ settings.isContentAnalysisActive && <SidebarItem key="readability-analysis" renderPriority={ 10 }>
+				<ReadabilityAnalysis />
+			</SidebarItem> }
+			{ settings.isKeywordAnalysisActive && <SidebarItem key="seo-analysis" renderPriority={ 20 }>
+				<SeoAnalysis
+					resultsContainer={ <SeoResultsContainer as={ ContentAnalysis } /> }
+					shouldUpsell={ settings.shouldUpsell }
+					shouldUpsellWordFormRecognition={ settings.isWordFormRecognitionActive }
+				/>
+			</SidebarItem> }
+			{ settings.isCornerstoneActive && <SidebarItem key="cornerstone" renderPriority={ 30 }>
+				<CollapsibleCornerstone />
```

Comment:
The cornerstone toggle data lives in the SEO store too. If not fixed here, could you please create a task for this so we don't forget?